app/(auth)/forgot-password/head.tsx

```diff
@@ -0,0 +1,5 @@
+const Head = () => {
+  return <></>
```

Comment:
You can't just return an empty component for Head. 

Reference: https://beta.nextjs.org/docs/api-reference/file-conventions/head#supported-head-tags